packages/desktop/views/dashboard/account-management/AccountManagement.svelte

```diff
@@ -9,12 +53,59 @@
+                                        <ClickableTile onClick={handleAccountClick}>
+                                            <div class="flex flex-col space-y-1">
+                                                <div class="flex space-x-2">
+                                                    <Text
+                                                        type={TextType.h5}
+                                                        color="gray-800"
+                                                        darkColor="gray-500"
+                                                        fontWeight={FontWeight.semibold}
+                                                    >
+                                                        {localize('views.accountManagement.list.tile.title')}
+                                                        {index + 1}
+                                                    </Text>
+                                                    {#if isAnAccount(account)}
+                                                        <Pill backgroundColor="blue-200" textColor="blue-600"
+                                                            >{localize(
+                                                                'views.accountManagement.list.tile.pill.main'
+                                                            )}</Pill
+                                                        >
+                                                    {/if}
+                                                    {#if isAnImplicitAccount(account)}
+                                                        <Pill backgroundColor="yellow-200" textColor="yellow-900"
+                                                            >{localize(
+                                                                'views.accountManagement.list.tile.pill.pending'
+                                                            )}</Pill
+                                                        >
+                                                    {/if}
+                                                </div>
+                                                <Text
+                                                    type={TextType.p}
+                                                    fontSize="12"
+                                                    lineHeight="leading-140"
+                                                    color="gray-600">{formatAndTruncateAccount(account.output)}</Text
+                                                >
+                                            </div>
+                                        </ClickableTile>
+                                    {/each}
+                                </list-wrapper>
+                            </left-pane-container>
+                        </Pane>
+                    </left-pane>
+                {/if}
+                {#if features.accountManagement.accountDetails.enabled}
```

Comment:
I would put the `if` of the `feature` inside the pane so that the view does not go out of alignment